### PR TITLE
deploy on tag

### PR DESCRIPTION
**Pull request by thitu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,17 +18,32 @@ install:
...
 jobs:
   include:
   - name: build
     stage: build
     script: make
-  - name: package debian
+  - name: create .DEB package
     stage: package
-    script: sudo -H debian/package-builder.sh 1.0.0
+    script: sudo -H debian/deb-builder.sh 0.0.1-ci
```
Comment (jmtrusona) [inline .travis.yml:29]: Hardcoded version `0.0.1-ci`

**Review by jmtrusona — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -18,16 +18,12 @@ install:
   - sudo -H ci/dependencies.sh > /dev/null
 
 stages:
-  - build
-  - test
+  - test # travis magic because of compiler matrix
```
Comment (jmtrusona) [inline .travis.yml:21]: 🧙

File: .travis.yml (hunk 1 above)
Comment (thitu) [reply .travis.yml:29]: yes .. for CI purposes

**Review by jmtrusona — APPROVED** [review]

File: debian/deb-builder.sh [hunk 3]
```diff
@@ -4,6 +4,10 @@ set -e
 
 version=$1
```
Comment (jmtrusona) [inline debian/deb-builder.sh:5]: One liner is:

```
version=${$1:-0.0.1-unspecified}
```

Comment (jmtrusona) [reply debian/deb-builder.sh:5]: I think right?

**Pull request merged**